docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.1"
+services:
+  ucurtma-container:
+    build:
+      context: ./
+      dockerfile: ./docker/alpine/Dockerfile
+    container_name: ucurtma-container
+    environment:
+      - BROWSERSTACK_USERNAME=0
```

Comment:
is it possible to use a `.env` file for enforcing the environment variables here?